specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -163,40 +163,33 @@
       "description": "The created subscription object.",
       "properties": {
         "subscriptionLink": {
-          "description": "The link to the new subscription.",
+          "description": "The link to the new subscription. Use this link to check the status of subscription creation operation.",
           "type": "string"
         }
       }
     },
     "SubscriptionCreationParameters": {
-      "description": "Subscription Creation Parameters required to create a new Azure subscription.",
+      "description": "The parameters required to create a new subscription.",
```

Comment:
Please, add `"required": [ …. ]` field and also `"type": "object"`.

Reply:
Added required field